cmd/optimizely/testdata/default.yaml

```diff
@@ -32,6 +32,10 @@ client:
   datafileURLTemplate: "https://localhost/v1/%s.json"
   eventURL: "https://logx.localhost.com/v1"
   sdkKeyRegex: "custom-regex"
+  datafileCacheService:
```

Comment:
datafileCacheServices will be more relevant.

Reply:
This is to keep it similar to the naming convention we are following for `userProfileService`.